src/app/dataset-view/additional-components/metadata-component/components/form-components/json-kind-field/json-kind-field.component.html

```diff
@@ -0,0 +1,31 @@
+<div [formGroup]="form">
+    <div class="d-flex">
+        <app-tooltip-icon [tooltip]="tooltip"></app-tooltip-icon>
+        <label class="form-label">{{ label }} </label>
+    </div>
+    <div class="ms-4" *ngFor="let control of data">
+        <div class="form-check">
+            <input
+                class="form-check-input"
+                type="radio"
+                [formControlName]="controlName"
+                [name]="controlName"
+                type="radio"
+                [value]="control.value"
+            />
+            <label class="form-check-label" for=""> {{ control.label }} </label>
+        </div>
+    </div>
+    <ng-container *ngIf="currentJsonKind === readKind.JSON">
```

Comment:
There is no need for the ng-container, you could use in on the <div>

Reply:
Removed